### PR TITLE
Add Adusumilli et al. (2020) melt rates

### PR DESCRIPTION
**Pull request by xylar**
This merge switches climatology maps of Antarctic melt rates to use Adusumilli et al. (2020) instead of Rignot et al. (2013).  It also adds the Adusumilli et al. melt rates and fluxes to the time-series plots as a 3rd observational point (keeping Rignot et al. 2013 present-day and steady-state estimates).

To do:
- [x] run test suite
- [x] run for SORRM simulation
- [x] update documentation

### CODE REVIEW

**Comment by xylar** [comment]
## Testing

I ran the test suite on Perlmutter.  Results can be found here:
https://portal.nersc.gov/cfs/e3sm/xylar/analysis_testing/pm-cpu/add-adusumilli-melt-rates/

In particular, the Adusumilli et al. (2020) melt rates can be seen in both climatology maps:
image
and time series:
image

**Review by cbegeman — COMMENTED** [review]
@xylar This all looks great! What testing would you like from me?

**Comment by xylar** [comment]
@cbegeman, good question.  If you have time, you could run MPAS-Analysis using this branch on a decade of a recent SORRM simulation.

I plan to use this branch on the 3 runs for the paper (I just haven't got to it since LCRC came back up.  Once that's done, I'll ask you to have a look at that, too.

File: preprocess_observations/preprocess_adusumilli_melt.py [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+        ds_out.meltRate.attrs = melt_attrs
+        mask = ds_out.uncertMask > 0.
+        ds_out['meltRateUncertainty'] = \
+            (np.sqrt(ds_out.meltRateUncertSqr)).where(mask)
+        ds_out.meltRateUncertainty.attrs = uncert_attrs
+        ds_out = ds_out.drop_vars(['meltRateUncertSqr'])
+        ds_out.to_netcdf(f'{out_prefix}_{out_grid_name}.{date}.nc')
+
+
+def compute_regional_means(in_filename, out_prefix, date, chunk_size=50000,
```
Comment (xylar) [inline preprocess_observations/preprocess_adusumilli_melt.py:184]: @cbegeman, here's the function I would appreciate you sanity-checking if you have time.

**Comment by xylar** [comment]
Here is the analysis from a 5-year run on SORRM that used prescribed melt rates from Adusumilli, which makes for a nice sanity check both on the analysis and on the prescribed melt fluxes:
https://web.lcrc.anl.gov/public/e3sm/diagnostic_output/ac.xasay-davis/analysis/20230516.GMPAS-IAF-DISMF.T62_SOwISC12to60E2r4.chrysalis/

Melt fields are nearly identical, with small differences due to interpolation artifacts:
image

Regionally averaged melt time series are similar but slightly smaller from MPAS-Ocean, e.g.:
image

**Comment by xylar** [comment]
@cbegeman, if the analysis I've done so far is sufficient for you to be happy, I don't think you need to do anything else in your review except give it a look.  (A second pair of eyes on the regional preprocessing was the main thing I wanted.)

**Comment by milenaveneziani** [comment]
Looks great to me @xylar.

Sort of unrelated question: what does the 'SS' stand for in one of the Rignot's labels?

**Review by milenaveneziani — APPROVED** [review]
Approved based on visual inspection and developer testing.

**Comment by xylar** [comment]
> Sort of unrelated question: what does the 'SS' stand for in one of the Rignot's labels?

Steady state.  This would be the melt rates if the ice sheet were in equilibrium instead of losing mass.  (So an approximation of what melt rates might have been in preindustrial times.)

**Comment by xylar** [comment]
Thanks @milenaveneziani!

**Pull request merged**